### DISCUSSION TITLE
Disable microphones but keep line in?

### DISCUSSION THREAD

**Discussion by greno** [pschatzmann/arduino-audiokit#64] [Q&A] [answered]
I think I read this somewhere, but now I can't find it.  Is it true that for the ai_thinker (ES8388), you can't disable the microphone inputs without also disabling the line in inputs? I'm working on a guitar effects pedal, but I'm getting feedback through the microphones.

If I was going to physically modify the board to disable the microphones, what would be the best way to do it? Just unsolder the microphones?

**Comment by boondockadmin** [comment, marked as answer]
Hi Greno, I had a similar problem with Audiokit. I ended up desoldering the Microphones. To the best of my knowledge, changing INPUT in Audiokit won't help. Only option is to desolder the Mics.

**Reply by greno** [reply]
OK - thanks for the confirmation!
